test/utils/Processing.test.ts

```diff
@@ -459,13 +468,30 @@ describe('Processing', () => {
               tempIGPubRoot,
               'fsh-generated',
               'resources',
-              'StructureDefinition-my-duplicate-profile.json'
+              'Patient-my-duplicate-profile.json'
```

Comment:
Shouldn't this _still_ be `StructureDefinition-my-duplicate-profile.json`?  You didn't make any changes to profile-related code or test fixtures -- and there is nothing in the predefined resources or FSH resources that _would_ make a file called `Patient-my-duplicate-profile.json` anyway -- so you're not really testing anything as it currently is.  If it's `StructureDefinition-my-duplicate-profile.json` though, then you're at least testing that the FSH profile was not created.

Reply:
Fixed in 5917ef7